data/changelog.txt

```diff
@@ -227,43 +249,46 @@ Major optimization of field processing.
 Adds reachability cache/filter that can reject bresenham-based LOS checks in O(1) time.
 Uses field_cache in map::get_field to speedup fire check in npc::assess_danger.
 Speedup quicksave by limiting ui redraw rate
+Prevents slow processing of NPC overmap travel when `goal` is unreachable.
+Fixes the performance bug caused by `verify_vehicle_cache` recalculation.
+Adds lightweight crafting inventory to improve NPC construction time.
 
 ## Infrastructure:
 Adds basic infrastructure for diseases.
 Allow for specialization of activity parameters.
 Improved event transformations and statistic tracking.
 Jsonize Autodoc quality.
-Add tests for character BMI / kcal / weight.
+Adds tests for character BMI / kcal / weight.
 Jsonize body parts.
 Jsonize mood modifier from bodyparts.
 Allow multiple overmap terrains in start location.
 Make test failures from other platforms more easily reproducible.
 Enforce a standard header guard format.
```

Comment:
```suggestion
Jsonizes body parts.
Jsonizes mood modifier from body part conditions (e.g., wet).
Allows multiple different overmap terrains for start locations.
Makes test failures from other platforms more easily reproducible.
```
I am not sure how much "Enforces a standard header guard format" is going to be of interest.

Reply:
Agreed.